can/interfaces/kvaser/canlib.py

```diff
@@ -319,6 +319,21 @@ def __check_bus_handle_validity(handle, function, arguments):
         errcheck=__check_status_operation,
     )
 
+    kvFlashLeds = __get_canlib_function(
+        "kvFlashLeds",
+        argtypes=[c_canHandle, ctypes.c_int, ctypes.c_int],
+        restype=ctypes.c_short,
+        errcheck=__check_status,
+    )
+
+    if sys.platform == "win32":
+        canGetVersionEx = __get_canlib_function(
+            "canGetVersionEx",
+            argtypes=[ctypes.c_uint],
+            restype=ctypes.c_uint,
+            errcheck=__check_status,
+        )
+
```

Comment:
In what way are these changes related to IXXAT?

Reply:
My bad, I reverted commit ad11bf95 that introduced the (unwanted) change.